x-pack/plugins/apm/server/routes/services/get_services/get_services_from_error_and_metric_documents.ts

```diff
@@ -37,7 +37,7 @@ export async function getServicesFromMetricDocuments({
     'get_services_from_metric_documents',
```

Comment:
We could also add `error` to the operationName.

Reply:
good catch, updated!

Reply:
nice one, thanks!